### ISSUE TITLE
`on_epoch_end` in Callback doesn't receive `acc` key in `logs` argument anymore?

### ISSUE THREAD

**Issue by iesahin** [fchollet/keras#2326]
Hi,

I was using the following piece to report the training progress

```
class SlackerProgress(Callback):                                                                                            
    def on_epoch_end(self, epoch, logs={}):                                                                                 
        hostname = socket.gethostname()                                                                                     
        msg = """End of Epoch {} in {}                                                                                      
                                                                                                                            
*loss:*  {:.2f}                                                                                                             
*acc:* {:.2f}                                                                                                               
*val_loss:* {:.2f}                                                                                                          
*val_acc:* {:.2f}                                                                                                           
                                                                                                                            
""".format(epoch, hostname,                                                                                                 
           logs["loss"], logs["acc"], logs["val_loss"], logs["val_acc"])                                                    
        slacker_message(msg)                                                                                                

```                                                                   

This was working before, but after upgrading to latest `master`, it looks `logs["acc"]` isn't supplied anymore. It gives a KeyError. (I suspect it's related with `show_accuracy` being deprecated.)

It looks documentation in http://keras.io/callbacks/ should be updated with the keys as well. 

Thank you very much.

**Comment by iesahin** [comment]
Adding `metrics=["accuracy"]`to `model.compile` solves this problem but the docs should reflect this IMO.

**Comment by fchollet** [comment]
And so do they.
